server/jvm/src/test/scala/org/http4s/server/staticcontent/StaticContentSharedJvm.scala

```diff
@@ -18,40 +18,15 @@ package org.http4s
 package server
 package staticcontent
 
-import cats.effect.IO
 import fs2._
-import org.http4s.syntax.all._
 import org.http4s.testing.AutoCloseableResource
 
 import java.nio.charset.StandardCharsets
 import java.nio.file.Files
 import java.nio.file.Paths
 
-private[staticcontent] trait StaticContentShared { this: Http4sSuite =>
-  def routes: HttpRoutes[IO]
-
-  lazy val testResource: Chunk[Byte] = {
-    val s = getClass.getResourceAsStream("/testresource.txt")
-    require(s != null, "Couldn't acquire resource!")
-    val bytes =
-      AutoCloseableResource.resource(
-        scala.io.Source
-          .fromInputStream(s)
-      )(
-        _.mkString
-          .getBytes(StandardCharsets.UTF_8)
-      )
-
-    Chunk.array(bytes)
-  }
-
-  lazy val testResourceGzipped: Chunk[Byte] = {
-    val url = getClass.getResource("/testresource.txt.gz")
-    require(url != null, "Couldn't acquire resource!")
-    val bytes = Files.readAllBytes(Paths.get(url.toURI))
-
-    Chunk.array(bytes)
-  }
+private[staticcontent] trait StaticContentSharedJvm extends StaticContentShared {
```

Comment:
Was splitting this file necessary to avoid linking errors? It's not a huge deal, but as long as we are still merging from uncrossed series/0.22 into series/0.23 I've been trying to avoid this.

Reply:
Lemme take a look. I think I should be able to put it all back together.

Reply:
So it worked. I moved the jvm related stuff into `shared`, but as it's not referenced from the tests in `shared` – linker didn't yell.